orcestra/sat.py

```diff
@@ -112,3 +113,43 @@ def get_track_for_day(self, day):
             df["time"], format="%d %b %Y %H:%M:%S.000", exact=False
         )
         return df.set_index("time").sort_index().to_xarray()
+
+
+class SattrackLoader:
+    def __init__(self, satelite_name, forecast_day):
+        self.satelite_name = satelite_name
```

Comment:
It took me some digging in other repos to find out which satellite name is actually supported... I would at least add a doc string.

Reply:
Yes, indeed. In the long run, there should likely be an index which would list possible valid values. However this doesn't exist for now and thus I'd keep it with the docstring.